<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiningPage</class>
 <widget class="QWidget" name="MiningPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mining</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <item row="4" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelPercentHR">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="4">
    <widget class="QCheckBox" name="pokCheckBox">
     <property name="text">
      <string>Use Proof of Knowledge</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="threadsLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Percentage max hash rate:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QLineEdit" name="usernameLine">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLabel" name="usernameLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Username or Address</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="serverLine">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="portLine">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="value" stdset="0">
      <number>9332</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="serverLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Pool Server URL</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="portLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QLabel" name="passwordLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Password*</string>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QLineEdit" name="passwordLine">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhHiddenText</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="typeBox">
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Solo Mining</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pool Mining</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="startButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #15444A;</string>
     </property>
     <property name="text">
      <string>Start Mining</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="5">
    <widget class="Line" name="line">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="14" column="4">
    <widget class="QCheckBox" name="debugCheckBox">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Debug</string>
     </property>
    </widget>
   </item>
   <item row="14" column="5">
    <widget class="QPushButton" name="clearButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="5">
    <widget class="QListWidget" name="list">
     <property name="font">
      <font>
       <family>Monaco</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">margin-bottom: 10px;</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="autoScrollMargin">
      <number>100000</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragOnly</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="movement">
      <enum>QListView::Snap</enum>
     </property>
     <property name="batchSize">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QLabel" name="mineSpeedLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QLabel" name="networkHashRate">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="13" column="1" colspan="5">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="4" colspan="2">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>15</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>* Password not required on NOMP Pools</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>typeBox</tabstop>
  <tabstop>serverLine</tabstop>
  <tabstop>portLine</tabstop>
  <tabstop>usernameLine</tabstop>
  <tabstop>passwordLine</tabstop>
  <tabstop>list</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
